src/main/java/io/leonis/subra/gui/field/Robots.java

```diff
@@ -0,0 +1,39 @@
+package io.leonis.subra.gui.field;
+
+import com.googlecode.lanterna.TerminalSize;
+import com.googlecode.lanterna.TextColor.*;
+import com.googlecode.lanterna.gui2.*;
+import io.leonis.subra.game.data.*;
+import java.util.Set;
+import lombok.AllArgsConstructor;
+
+/**
+ * @author Jeroen de Jong
+ */
+@AllArgsConstructor
+public class Robots extends AbstractComponent<Robots> {
```

Comment:
This should be `RobotPositions` or maybe even `RobotPositionsComponent` :')

Reply:
i'd prefer suffixing with `Component`